src/main/java/core/basesyntax/service/impl/FruitServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperationHandler;
+import core.basesyntax.service.FruitService;
+import java.util.List;
+import java.util.Map;
+
+public class FruitServiceImpl implements FruitService {
+    private static final String COLUMNS_NAME = "fruit,quantity";
+    private static final String CSV_SEPARATOR = ",";
+    Map<Operation, FruitOperationHandler> operations;
```

Comment:
```suggestion
    private Map<Operation, FruitOperationHandler> operations;
```
don't forget about modifiers